src/Common/Simplify.elm

```diff
@@ -19,16 +19,12 @@ ruleConfig : RuleConfig
 ruleConfig =
     { slug = Nothing
     , restrictToFiles = Nothing
-    , rules = [ ImportedRule (Simplify.rule Simplify.defaults) decodeSimplify ]
+    , rules = [ ImportedRule (Simplify.rule Simplify.defaults) simplifyDecoder ]
```

Comment:
What is this Simplify.defaults? I can't find the code for it anywhere (think I must be having a bad day :))

Reply:
This isn't from this repo, it's from [jfmengels/elm-review-simplify](https://package.elm-lang.org/packages/jfmengels/elm-review-simplify/latest/Simplify). Sorry the modules names are confusing. The defaults can be changed to allow a few very specific exceptions, but I don't think they apply here.